target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network

```diff
@@ -26,9 +29,30 @@ ipq60xx_setup_interfaces()
 	esac
 }
 
+ipq60xx_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
```

Comment:
What about this
```
ipq60xx_setup_macs()
{
	local board="$1"

	case "$board" in
		linksys,mr7350)
			hw_mac_addr=$(mtd_get_mac_ascii devinfo hw_mac_addr)
			ucidef_set_interface_macaddr "lan" "$hw_mac_addr"
			ucidef_set_interface_macaddr "wan" "$hw_mac_addr"
		;;
	esac
}
```
?

Reply:
It can be seen that two approaches are practiced:
- https://github.com/openwrt/openwrt/blob/main/target/linux/ipq40xx/base-files/etc/board.d/02_network#L157
- https://github.com/openwrt/openwrt/blob/main/target/linux/ipq806x/base-files/etc/board.d/02_network#L75

Reply:
@Vladdrako, the above looks much clearer.